spec/fixtures/output/k8s/single_image_error_pod.json

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
Did you manually update these files from existing files?  I think we should pull an existing json files from actual image error and crash loop pods (I'm surprised there isn't a crashloop json in here already?)

Reply:
I took an existing pod JSON and modified the relevant parts, ie the `status`.  I been testing with an app that doesn't use init containers and also has a long list of env variables and mounts so didn't want to create a massive file when the only part that matters is the status.  The status in the JSON file I added was copy+pasted from the actual output with kubectl, the other parts were just copied from existing JSON files.

Reply:
Should I regenerate these new JSON files will full contents copied from kubectl output? The files will be rather large unless I generate the errors with a slimmed down app that doesn't mount as many things.

Reply:
Yes please - I'd think it's just safer.  The size I'm not super worried about, I'd have to check the gemspec, but I don't believe test/ directory gets put into the gem (again I'd have to check to be sure).

In any case - yes I think real kubectl output is best.

Reply:
Added full JSON for the 2 test cases added in this pull request.